packages/libs/react-ui/src/styles/tokens/light.css.ts

```diff
@@ -0,0 +1,685 @@
+            n40: tokens.kda.foundation.color.palette.yellow.n40,
+            n50: tokens.kda.foundation.color.palette.yellow.n50,
+            n60: tokens.kda.foundation.color.palette.yellow.n60,
+            n70: tokens.kda.foundation.color.palette.yellow.n70,
+            n80: tokens.kda.foundation.color.palette.yellow.n80,
+            n90: tokens.kda.foundation.color.palette.yellow.n90,
+            n95: tokens.kda.foundation.color.palette.yellow.n95,
+            n99: tokens.kda.foundation.color.palette.yellow.n99,
+            n100: tokens.kda.foundation.color.palette.yellow.n100,
+          },
+        },
+        categorical: {
+          category1: {
+            default: '#2898bd',
+            '@hover': '#227d9b',
+          },
+          category2: {
+            default: '#5e4db2',
+            '@hover': '#352c63',
+          },
+          category3: {
+            default: '#e56910',
+            '@hover': '#c25100',
+          },
+          category4: {
+            default: '#943d73',
+            '@hover': '#50253f',
+          },
+          category5: {
+            default: '#09326c',
+            '@hover': '#1c2b41',
+          },
+          category6: {
+            default: '#8f7ee7',
+            '@hover': '#8270db',
+          },
+          category7: {
+            default: '#50253f',
+            '@hover': '#3d2232',
+          },
+          category8: {
+            default: '#a54800',
+            '@hover': '#702e00',
+          },
+        },
+      },
+      effect: {
+        defaultBlur: '',
+        blur: {
+          '0': '',
+        },
+        defaultShadow: `4px 0.5rem ${tokens.kda.foundation.size.n3} 4px #000000`,
+        shadow: {
+          '0': `4px 0.5rem ${tokens.kda.foundation.size.n3} 4px #000000`,
+          '1': `4rem 4rem ${tokens.kda.foundation.size.n3} 2rem #000000`,
+          '2': `0 0 ${tokens.kda.foundation.size.n3} 24px #000000`,
+        },
+      },
+      typography: {
+        family: {
+          primaryFont: 'Haas Grot Disp',
+          headingFont: 'Haas Grot Disp',
+          codeFont: 'Kode Mono',
+          bodyFont: 'Haas Grot Disp',
+        },
+        fontSize: {
+          xs: '0.75rem',
+          sm: '0.875rem',
+          base: tokens.kda.foundation.size.n4,
+          md: '1.125rem',
+          lg: '1.25rem',
+          xl: '1.5rem',
+          '2xl': '1.75rem',
+          '3xl': '2rem',
+          '4xl': '2.25rem',
+          '5xl': '2.5rem',
+          '6xl': '2.75rem',
+          '7xl': '3rem',
+          '8xl': '3.25rem',
+          '9xl': '3.75rem',
+          '10xl': '4.5rem',
+          '11xl': '5rem',
+          '12xl': '5.25rem',
+        },
+        lineHeight: {
+          primaryFont: {
+            initial: '',
+            normal: '400',
+            base: '1.4',
+            lg: '1.9',
+          },
+          secondaryFont: {
+            initial: '',
+            normal: '400',
+            base: '1.4',
+            lg: '1.9',
+          },
```

Comment:
What differentiates the two?

Reply:
Question for isa i think ? But this could be answered outside of the MR

Reply:
yes, I'll go over them with Isa when setting up sprinkles 👍